File: Clients/NotatnikMechanika.Flutter/lib/features/domain/entities/car/CarModel.dart
import 'package:equatable/equatable.dart';

class CarModel extends Equatable {
  final int id;
  final String brand;
  final String plate; //Rejestracja
  final String engine;
  final String power;
  final String vin;
  final int customerId;

  CarModel({
    this.id,
    this.brand,
    this.plate,
    this.engine,
    this.power,
    this.vin,
    this.customerId,
  }) : super([id, brand, plate, engine, power, vin, customerId]);
}

File: Api/Shared/Models/Car/CarModel.cs
using System.ComponentModel.DataAnnotations;

namespace NotatnikMechanika.Shared.Models.Car
{
    public class CarModel : ValidateModelBase
    {
        public int Id { get; set; }
        [Required]
        public string Brand { get; set; }
        public string Model { get; set; }
        public string Plate { get; set; } //Rejestracja 
        public string Engine { get; set; }
        public string Power { get; set; }
        public string Vin { get; set; }
        public int CustomerId { get; set; }
    }
}
